src/Catalyst.Core.Modules.Dfs/Controllers/V0/NameController.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Dfs.CoreApi;
+using Catalyst.Core.Lib.Util;
+using Lib.P2P;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Catalyst.Core.Modules.Dfs.Controllers.V0
+{
+    /// <summary>
+    ///   Content that has an associated name.
+    /// </summary>
+    public class NamedContentDto
+    {
+        /// <summary>
+        ///   Path to the name, "/ipns/...".
+        /// </summary>
+        public string Name;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)